### PR TITLE
website/integrations: add documentation for OIDC setup with Paperless-ngx

### PR DESCRIPTION
**Pull request by girlpunk**
## Details

Add documentation for setting up OIDC authntication for Paperless-ngx

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [X] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by rissson** [comment]
Heya, thanks for your contribution! You also need to add the new documentation to `website/sidebarIntegrations.js`. I'll be going through migrating my paperless-ngx to using OIDC soon-ish, I'll review this then.

File: website/integrations/services/paperless-ngx/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Paperless-ngx
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Paperless-ngx
+
+> Paperless-ngx is an application that indexes your scanned documents and allows you to easily search for documents and store metadata alongside your documents. It was a fork from paperless-ngx, in turn a form from the original Paperless, both of which are no longer maintained.
```
Comment (tanberry) [inline website/integrations/services/paperless-ngx/index.md:9]:
```suggestion
> Paperless-ngx is an application that indexes your scanned documents and allows you to easily search for documents and store metadata alongside your documents. It was a fork from paperless-ngx, in turn a fork from the original Paperless, neither of which are maintained any longer.
```

File: website/integrations/services/paperless-ngx/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+> -- https://github.com/paperless-ngx/paperless-ngx
+
+## Preparation
+
+The following placeholders will be used:
+
+- `paperless.company` is the FQDN of the Paperless-ngx install.
+- `authentik.company` is the FQDN of the authentik install.
+
+## Authentik Configuration
```
Comment (tanberry) [inline website/integrations/services/paperless-ngx/index.md:20]:
```suggestion
## authentik Configuration
```

File: website/integrations/services/paperless-ngx/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### Step 2 - Application
+
+Create an application (under Resources/Applications) with these settings:
+
+    Name: Paperless
+    Slug: paperless
+    Provider: Paperless
+
+## Paperless
```
Comment (tanberry) [inline website/integrations/services/paperless-ngx/index.md:37]:
```suggestion
## Paperless Configuration
```

File: website/integrations/services/paperless-ngx/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Add the following environment variables to your Paperless-ngx setup. If you are using docker-compose, then add the following to your docker-compose.yml file:
```
Comment (tanberry) [inline website/integrations/services/paperless-ngx/index.md:39]:
```suggestion
Add the following environment variables to your Paperless-ngx setup. If you are using Docker Compose, then add the following to your docker-compose.yml file:
```

File: website/integrations/services/paperless-ngx/index.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+Now restart your container:
+`docker compose down && docker compose up -d`
+
+## Finished
+
+Now you can access Paperless-ngx by logging in with authentik.
+
+To add Authentik authentication to an existing user, log in to Paperless with local authentication, click the profile icon in the top-right, click My Profile, then Connect new social account.
```
Comment (tanberry) [inline website/integrations/services/paperless-ngx/index.md:69]:
```suggestion
To add authentik authentication to an existing user, log in to Paperless with local authentication, click the profile icon in the top-right, click My Profile, then Connect new social account.
```

**Review by tanberry — COMMENTED** [review]
Thanks so much @girlpunk for this contribution. I made a few word edits, @rissson will do the technical review. We appreciate your contribution!

File: website/integrations/services/paperless-ngx/index.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  {
+    "openid_connect": {
+      "APPS": [
+        {
+          "provider_id": "authentik",
+          "name": "Authentik",
+          "client_id": "< Client ID >",
+          "secret": "< Client Secret >,
+          "settings": {
+            "server_url": "https://authentik.company/application/o/authorize/"
```
Comment (rissson) [inline website/integrations/services/paperless-ngx/index.md:53]:
```suggestion
            "server_url": "https://authentik.company/application/o/paperless/.well-known/openid-configuration"
```

**Review by rissson — CHANGES_REQUESTED** [review]

**Review by rissson — APPROVED** [review]

**Pull request merged**